components/clarinet-files/src/lib.rs

```diff
@@ -33,16 +34,13 @@ pub const DEFAULT_DEVNET_BALANCE: u64 = 100_000_000_000_000;
 pub type FileAccessorResult<T> = Pin<Box<dyn Future<Output = Result<T, String>>>>;
 
 pub trait FileAccessor {
-    fn file_exists(&self, location: FileLocation) -> FileAccessorResult<bool>;
-    fn read_manifest_content(
+    fn file_exists(&self, path: String) -> FileAccessorResult<bool>;
+    fn read_file(&self, path: String) -> FileAccessorResult<String>;
```

Comment:
could we use references `&str` instead of `String`? That'll helps us avoid cloning.